scripts/repl.js

```diff
@@ -204,6 +204,29 @@
     evt.stopPropagation();
   }
 
+  function envItemToString(item) {
+    return item.name + ' ' + JSON.stringify(item.targets);
+  }
+
+  function onEnvBuild(node, opts, envResult) {
```

Comment:
@existentialism Yeah. I think there would be a couple of changes after 2.0 merge. So we'll keep our fingers on the pulse. 

>We might want to expose the debug output from directly from the plugin on onPresetBuild instead of needing to reconstruct it from structured data?

Initially, I've chosen this way, but decided that we could come up with awesome UI idea to display plugins/built-ins neat (maybe in the table or with the better colorful console). 🌅